### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by millerlarson** [raiguard/Factorio-SmallMods#3]
I love the functionality of this little mod and _really_ wish it could be integrated into the GUI...  So thank you for this, it fills a need for sure.  Especially with the temp logistic requests!

Factorio: 0.18.2
Quick Item Search: 1.2.1

"You can navigate the interface completely with keyboard shortcuts..."

I'm struggling however, to understand the intended use with keyboard navigation.  I might have something else conflicting with keyboard shortcuts, etc... but after invoking the GUI, and typing in a string, I'm unable to find a way to switch to the search results and navigate with arrow keys.  If I just begin navigating with arrows, I'm stuck moving the cursor around the text input search field.  I can't seem to tab to the search results, click in a blank space, etc.  In short, arrow key navigation doesn't seem to be working for me.

Keyboard shortcuts for THIS mod are default.  Unsure if anything else could be conflicting, but I'm not sure what other information to provide; full mod list, savegame, etc.

Please let me know what would help if anything. Though, I'm hoping it could just be some explanation that is needed.

Thanks again for all your modding work!

**Comment by raiguard** [comment]
After typing your query, press enter. Use the arrow keys to navigate, then press enter again to confirm. I'll update the mod description with a better explanation.

Let me know if you have any other questions!

**Comment by millerlarson** [comment]
Yep.  I'm an idiot.

Thanks again!
